Server/VikingVault.DataAccess/VikingVaultDbContext.cs

```diff
@@ -31,10 +33,31 @@ public void SeedAdmin(ModelBuilder modelBuilder)
                     PictureLink = "",
                     Address = "Ioan Budai Deleanu Street 64",
                     Cnp = "1700820642466",
-                    Role = "admin"
+                    RoleId = 1
                 });
         }
 
+        public void SeedRoles(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<Role>().
+                HasData(new
+                {
+                    Id = 1,
```

Comment:
It make more sense to assign int values to the enums (because that's what an enum basically is) and then use those as the Id values
`enum RoleEnum { Admin = 1}` `Id = (int)RoleEnum.Admin`